### PR TITLE
[Doc] Fix issues in Upgrade and Downgrade

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
Signed-off-by: EsoragotoSpirit <wanglichen@starrocks.com>

Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

File: docs/deployment/downgrade.md [hunk 1]
```diff
@@ -35,15 +35,35 @@ During preparation, you must perform the compatibility configuration if you are
...
+Before downgrading your StarRocks cluster, you must disable tablet clone.
+
+```SQL
+ADMIN SET FRONTEND CONFIG ("max_scheduling_tablets" = "0");
+ADMIN SET FRONTEND CONFIG ("max_balancing_tablets" = "0");
+ADMIN SET FRONTEND CONFIG ("disable_balance"="true");
+ADMIN SET FRONTEND CONFIG ("disable_colocate_balance"="true");
+```
+
+After the downgrade, and the status of all BE nodes is `Alive`, you can re-enable tablet clone.
```
Comment (evelynzhaojie) [inline docs/deployment/downgrade.md:51]: After the downgrade, you can enable tablet clone again if the status of all BE nodes becomes `Alive`.

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Pull request merged**